src/app/builder/episode/episode-picker.component.ts

```diff
@@ -39,10 +39,22 @@ export class EpisodePickerComponent implements OnChanges, OnInit {
     this.select.emit(episode);
   }
 
+  private proxiedFeedUri() {
+    const customProxyUrl= window['ENV']['FEED_PROXY_URL'];
```

Comment:
this feels like it could be const `PROXY_URL=window['ENV']...||"/proxy?url="` and be hoisted to the top of the file so this doesn't execute every time, though that's I think a matter of personal taste.